### ISSUE TITLE
`collapse = TRUE` and Quarto support

### ISSUE THREAD

**Issue by cderv** [yihui/knitr#2043]
I just saw that 
````markdown
---
title: "Title"
format: 
  html: default
---

```{r collapse = TRUE}
1 + 1
2 + 2
3 + 3
```
````

does not work as expected with Quarto 
![image](https://user-images.githubusercontent.com/6791940/134034933-8c2075c9-6144-4a73-b393-dd763c872b28.png)

With rmarkdown we get 
![image](https://user-images.githubusercontent.com/6791940/134035051-7f3cf3ae-7320-4ea7-9337-19877a9c61ab.png)

I think it is a matter of teaching **knitr** about Quarto HTML organization because it is organized using some div we don't get by Pandoc only, specifically `<div class="cell-output-stdout">`

````html
<div class="cell">
<div class="sourceCode" id="cb1"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="dv">1</span> <span class="sc">+</span> <span class="dv">1</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>[1] 2</code></pre>
</div>
<div class="sourceCode" id="cb3"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="dv">2</span> <span class="sc">+</span> <span class="dv">2</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>[1] 4</code></pre>
</div>
<div class="sourceCode" id="cb5"><pre class="sourceCode r cell-code code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="dv">3</span> <span class="sc">+</span> <span class="dv">3</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-stdout">
<pre><code>[1] 6</code></pre>
</div>
</div>
````

**Comment by jjallaire** [comment]
Yes, we should definitely support this. Will work on this soon!

**Comment by cderv** [comment]
Fixed in Quarto 0.2.163
https://github.com/quarto-dev/quarto-cli/commit/c6630fb601315517f48569ca19f1e0031ce05d68

We correctly get 
![image](https://user-images.githubusercontent.com/6791940/134382582-d379b7b3-c5ee-4ab7-a9de-2682dbfedb8e.png)
